scst-store/amr/graphql-query.hbs.md

```diff
@@ -3,15 +3,83 @@
 This topic tells you how to connect to the GraphQL playground and how to query each supported data model.
 For information about the AMR data models, see [AMR Data Models](data-model-and-concepts.hbs.md).
 
-## <a id='connecting-to-graphql'></a> Connecting to GraphQL playground
+## <a id='connecting-to-graphql'></a> Connecting to AMR GraphQL
+
+There are two ways you can perform GraphQL queries:
+
+- Using the GraphQL playground
+- Using [cURL](https://curl.se/)
 
 Vmware recommends enabling ingress. The Supply Chain Security
```

Comment:
Sorry, i know it's not your change but our docs should probably have the correct capitalization

`Vmware` -> `VMware`